controller/k8s_resources.py

```diff
@@ -67,8 +67,12 @@ def create_template_values(name, spec):
         "jupyter_server_cookie_secret": os.urandom(32).hex(),
         "name": name,
         "oidc": spec["auth"]["oidc"],
-        "path": os.path.join("/", spec["routing"]["path"].rstrip("/")),
-        "probe_path": os.path.join("/", spec["routing"]["path"].rstrip("/"), "api"),
+        "path": (
```

Comment:
What's the intention here? Are there cases where the missing trailing slash in the path was a problem?